db/migrate/20130709105614_add_editable_is_default_and_index_on_editable_is_default_and_name_to_spree_roles.rb

```diff
@@ -3,7 +3,6 @@ def change
     add_column :spree_roles, :editable, :boolean, :default => true
     add_column :spree_roles, :is_default, :boolean, :default => false
 
-    add_index(:spree_roles, :name)
```

Comment:
This index is now added by Spree itself. So, we are not removing it from DB. If a user is on a previous version, the index will be present and thus it won't affect that user. For new users, the index will be created by spree itself.